src/software/thunderscope/replay/proto_player.py

```diff
@@ -149,8 +148,11 @@ def unpack_log_entry(log_entry: str) -> (float, Type[Message], Message):
         except NameError:
             raise TypeError(f"Unknown proto type in replay: '{protobuf_type}'")
 
+        # filtered = data[:-len('\n')]
+        # print(f"{timestamp=}  {protobuf_type=}  {data=}  {filtered=}") # TODO (NIMA)
+
         # Deserialize protobuf
-        proto = proto_class.FromString(base64.b64decode(data[len("b") : -len("\n")]))
+        proto = proto_class.FromString(base64.b64decode(data[: -len("\n")]))
```

Comment:
This breaks backwards compatibility. I can manually add a `b` at the start of every serialized proto in the file, though that's a bit annoying. Long term, we should add some metadata somewhere in the replay files (e.g. first line) where we define the replay file version, so we can support different versions easily.